### DISCUSSION TITLE
Feature Request : Microsoft Teams integration

### DISCUSSION THREAD

**Discussion by thomas-alkaige** [dailydotdev/daily#1470] [Feature Requests]
Hey,

I really like the squad feature but I miss some integration with Microsoft Teams, the same way you have with Slack!
I searched to do it myself but I didn't found API so I'm kinda stuck 😢 

Thanks !

**Comment by idoshamun** [comment]
Hi @thomas-alkaige,

Unfortunately we don't have an API yet but we're considering to add teams support in the future 🙏
